quirks/change-password-URLs.json

```diff
@@ -24,5 +24,6 @@
     "twitch.tv": "https://www.twitch.tv/settings/security",
     "xfinity.com": "https://customer.xfinity.com/users/me/update-password",
     "yahoo.com": "https://login.yahoo.com/account/change-password",
-    "zoom.us": "https://zoom.us/profile#pwd-form"
+    "zoom.us": "https://zoom.us/profile#pwd-form",
+    "umsystem.edu": "https://password.umsystem.edu/reset/"
```

Comment:
This looks like a reset page. Is there a “change” page, or are changes done via reset?

Reply:
changes are done via reset, if you visit the page it prompts you for user/pass, but provides options if you don't know the password. 

they require some kind of 2FA to change password (which is not generally required), which is historically why it is called reset